pkg/task/common.go

```diff
@@ -7,9 +7,11 @@ import (
 	"crypto/tls"
 	"fmt"
 	"net/url"
+	filepath "path"
```

Comment:
??? there isn't another `path` package. calling it `filepath` is confusing since `path/filepath` package does exist.

Reply:
yes, my mistake, I didn't notice IDE add this package automatically...